imports/openai/install-package.ts

```diff
@@ -0,0 +1,128 @@
+import { DeepClient } from "@deep-foundation/deeplinks/imports/client";
+import * as fs from "fs";
+import {PACKAGE_NAME} from "./package-name";
+require('dotenv').config();
+
+export async function installOpenAiPackage({ deep }: { deep: DeepClient }) {
+    const anyTypeLinkId = await deep.id('@deep-foundation/core', "Any");
+    const userTypeLinkId = await deep.id('@deep-foundation/core', "User");
+    const typeTypeLinkId = await deep.id('@deep-foundation/core', "Type");
+    const syncTextFileTypeLinkId = await deep.id('@deep-foundation/core', "SyncTextFile");
+    const containTypeLinkId = await deep.id('@deep-foundation/core', "Contain");
+    const dockerSupportsJsLinkId = await deep.id('@deep-foundation/core', "dockerSupportsJs");
+    const handlerTypeLinkId = await deep.id('@deep-foundation/core', "Handler");
+    const handleInsertLinkId = await deep.id('@deep-foundation/core', "HandleInsert");
+    const packageTypeLinkId = await deep.id('@deep-foundation/core', "Package");
+    const joinTypeLinkId = await deep.id('@deep-foundation/core', "Join");
+    const { data: [{ id: packageLinkId }] } = await deep.insert({
+    type_id: packageTypeLinkId,
+    string: { data: { value: PACKAGE_NAME } },
+    in: { data: [
+      {
+        type_id: containTypeLinkId,
+        from_id: deep.linkId
+      },
+    ] },
+    out: { data: [
+      {
+        type_id: joinTypeLinkId,
+        to_id: await deep.id('deep', 'users', 'packages'),
+      },
+      {
+        type_id: joinTypeLinkId,
+        to_id: await deep.id('deep', 'admin'),
+      },
+    ] },
+  });
+  
+    const { data: [{ id: userInputLinkId }] } = await deep.insert({
+      type_id: syncTextFileTypeLinkId,
+      string: { data: { value: "user input" } },
+      in: {
+        data: {
+          type_id: containTypeLinkId,
+          from_id: deep.linkId,
+        },
+      },
+    });
```

Comment:
Move functionality related to a user to Buttons. Create them on pages/openai.tsx

Reply:
Do not forget to do this not only to these lines but in the code below